src/server/routes/actions.ts

```diff
@@ -19,6 +19,7 @@ router.get<unknown, BalanceResponse>("/balance", (_, res) => {
     .catch((e) => {
       logger.error(e);
       errorCounter.plusOne("other");
+      res.send({ balance: "0" });
```

Comment:
This endpoint doesn't have an error type of result, so following the existing pattern at the moment.